spiceaidocs/docs/components/data-connectors/odbc.md

```diff
@@ -42,6 +43,27 @@ datasets:
       odbc_connection_string: Driver={Foo Driver};Host=db.foo.net;Param=Value
 ```
 
+### Selecting SQL Dialect
+
+The default SQL dialect may not be supported by every ODBC connection. The `odbc_sql_dialect` allows overriding the selected SQL dialect for a specified connection.
+
+The runtime will attempt to detect the dialect to use for a connection based on the contents of `Driver=` in the `odbc_connection_string`. The runtime will usually detect the correct SQL dialect for the following connection types:
+
+* PostgreSQL
+* MySQL
+* SQLite
+* Databricks
+
+These connection types are also the supported values for overriding dialect in `odbc_sql_dialect`. For example, overriding the dialect for your connection to a `postgresql` style dialect:
```

Comment:
Can you have a list of the supported values here that are accepted by `sql_dialect`, similar to what you have above.